src/graphics/device.js

```diff
@@ -222,9 +212,26 @@ pc.extend(pc, function () {
 
         // Retrieve the WebGL context
         if (canvas)
-            this.gl = _createContext(canvas, options);
+            var preferWebGl2 = true;
```

Comment:
Please expose it as some option that is possible to pass to Device and Application.